pkg/proxy/ipvs/proxier.go

```diff
@@ -168,6 +168,9 @@ const sysctlForward = "net/ipv4/ip_forward"
 const sysctlArpIgnore = "net/ipv4/conf/all/arp_ignore"
 const sysctlArpAnnounce = "net/ipv4/conf/all/arp_announce"
 
+const ipvsServiceSchedulerAnnotation = "proxy.ipvs.alpha.k8s.io/scheduler"
```

Comment:
This one I can guess what it does but no idea what valid values are -- comments please